# update-manager's Finnish translation.
# Copyright (C) 2005-2006 Timo Jyrinki
# This file is distributed under the same license as the update-manager package.
# Timo Jyrinki <timo.jyrinki@iki.fi>, 2005-2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2010-04-02 07:16+0000\n"
"PO-Revision-Date: 2010-03-30 20:51+0000\n"
"Last-Translator: Tomi Juntunen <Unknown>\n"
"Language-Team: Finnish <ubuntu-fi@lists.ubuntu.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2010-04-07 08:08+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:336 ../UpdateManager/Core/utils.py:336
msgid "0 KB"
msgstr "0 kt"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:339 ../UpdateManager/Core/utils.py:339
msgid "1 KB"
msgstr "1 kt"

#. TRANSLATORS: download size of small updates, e.g. "250 KB"
#: ../DistUpgrade/utils.py:342 ../UpdateManager/Core/utils.py:342
#, python-format
msgid "%.0f KB"
msgstr "%.0f kt"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:345 ../UpdateManager/Core/utils.py:345
#, python-format
msgid "%.1f MB"
msgstr "%.1f Mt"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:208 ../DistUpgrade/distro.py:423
#, python-format
msgid "Server for %s"
msgstr "Palvelin maalle %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:226 ../DistUpgrade/distro.py:232
#: ../DistUpgrade/distro.py:248
msgid "Main server"
msgstr "Pääpalvelin"

#: ../DistUpgrade/distro.py:252
msgid "Custom servers"
msgstr "Määrittele palvelin"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:131
msgid "Could not calculate sources.list entry"
msgstr "sources.list-riviä ei voi laskea"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:234
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Pakettitiedostoja ei löydetty. Ehkä tämä ei ole Ubuntu-levy, tai kyseessä on "
"väärä arkkitehtuuri?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:277
msgid "Failed to add the CD"
msgstr "CD-levyä ei voitu lisätä"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:278
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"CD-levyä lisättäessä tapahtui virhe, päivitys keskeytyy. Tee tästä "
"virheraportti, jos kyseessä on toimiva Ubuntu-CD.\n"
"\n"
"Virheilmoitus oli:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:130
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Poista huonossa tilassa oleva paketti"
msgstr[1] "Poista huonossa tilassa olevat paketit"

#: ../DistUpgrade/DistUpgradeCache.py:133
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Paketti ”%s” on ristiriitaisessa tilassa ja se tulee asentaa uudelleen, "
"mutta sen pakettitiedostoa ei löydy. Haluatko poistaa tämän paketin nyt "
"jatkaaksesi?"
msgstr[1] ""
"Paketit ”%s” ovat ristiriitaisessa tilassa ja ne tulee asentaa uudelleen, "
"mutta niiden pakettitiedostoja ei löydy. Haluatko poistaa nämä paketit nyt "
"jatkaaksesi?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:231
msgid "The server may be overloaded"
msgstr "Palvelin saattaa olla ylikuormitettu"

#: ../DistUpgrade/DistUpgradeCache.py:341
msgid "Broken packages"
msgstr "Rikkinäisiä paketteja"

#: ../DistUpgrade/DistUpgradeCache.py:342
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Järjestelmä sisältää rikkinäisiä paketteja, joita ei voitu korjata tällä "
"ohjelmalla. Korjaa ne käyttämällä synapticia tai apt-get -komentoa ennen "
"jatkamista."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:621
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Päivitystä määriteltäessä tapahtui ratkaisematon ongelma:\n"
"%s\n"
"\n"
" Tämä voi johtua:\n"
" * Päivittämisestä Ubuntun julkaisemattomaan kehitysversioon\n"
" * Ubuntun tämänhetkisen kehitysversion käyttämisestä\n"
" * Ubuntuun kuulumattomien, epävirallisten ohjelmapakettien käyttämisestä\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:631
msgid "This is most likely a transient problem, please try again later."
msgstr "Tämä on luultavasti hetkellinen ongelma. Yritä myöhemmin uudelleen."

#: ../DistUpgrade/DistUpgradeCache.py:634
msgid ""
"If none of this applies, then please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"Jos kyse ei ole näistä, ilmoita tämä ohjelmavirhe paketille ”update-manager” "
"ja sisällytä kansion /var/log/dist-upgrade/ tiedostot virheraporttiin."

#: ../DistUpgrade/DistUpgradeCache.py:640
#: ../UpdateManager/UpdateManager.py:766
msgid "Could not calculate the upgrade"
msgstr "Tarvittavia päivitykseen liittyviä tarkistuksia ei voitu tehdä"

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid "Error authenticating some packages"
msgstr "Joitain paketteja todennettaessa tapahtui virhe"

#: ../DistUpgrade/DistUpgradeCache.py:691
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Joitain paketteja ei voitu todentaa. Tämä voi olla ohimenevä verkko-ongelma. "
"Voit yrittää myöhemmin uudelleen. Alla on luettelo todentamattomista "
"paketeista."

#: ../DistUpgrade/DistUpgradeCache.py:711
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Paketti \"%s\" on merkitty poistettavaksi, mutta se on poistojen "
"estolistalla."

#: ../DistUpgrade/DistUpgradeCache.py:715
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Pakollinen paketti \"%s\" on merkitty poistettavaksi."

#: ../DistUpgrade/DistUpgradeCache.py:724
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Paketista yritetään asentaa mustalla listalla olevaa versiota \"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:824
#, python-format
msgid "Can not mark '%s' for upgrade"
msgstr "Pakettia \"%s\" ei voi merkitä päivitettäväksi"

#: ../DistUpgrade/DistUpgradeCache.py:838
#, python-format
msgid "Can't install '%s'"
msgstr "Ei voitu asentaa pakettia \"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:839
msgid ""
"It was impossible to install a required package. Please report this as a "
"bug. "
msgstr ""
"Vaadittua pakettia ei voitu asentaa. Ole hyvä ja luo tästä virheraportti. "

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:848
msgid "Can't guess meta-package"
msgstr "Metapakettia ei voitu arvata"

#: ../DistUpgrade/DistUpgradeCache.py:849
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Järjestelmääsi ei ole asennettu ubuntu-desktop-, kubuntu-desktop-, xubuntu-"
"desktop- tai edubuntu-desktop-pakettia, joten käytössä olevaa Ubuntun "
"versiota ei voitu tunnistaa.\n"
"Asenna jokin luetelluista paketeista synapticilla tai apt-get-ohjelmalla "
"ennen jatkamista."

#: ../DistUpgrade/DistUpgradeController.py:86
msgid "Reading cache"
msgstr "Luetaan välimuistia"

#: ../DistUpgrade/DistUpgradeController.py:189
msgid "Unable to get exclusive lock"
msgstr "Ei saatu haluttua lukitusta"

#: ../DistUpgrade/DistUpgradeController.py:190
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Tämä tarkoittaa yleensä, että toinen pakettienhallintaohjelma (kuten apt-get "
"tai aptitude) on jo käynnissä. Sulje tämä toinen ohjelma ensin."

#: ../DistUpgrade/DistUpgradeController.py:236
msgid "Upgrading over remote connection not supported"
msgstr "Päivittäminen etäyhteyden yli ei ole tuettu"

#: ../DistUpgrade/DistUpgradeController.py:237
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Versiopäivitys ssh-yhteyden välityksellä ei ole tuettu. Jos haluat tehdä "
"versiopäivityksen ssh-yhteyden välityksellä, kirjoita komentoriville 'do-"
"release-upgrade'.\n"
"\n"
"Päivitys sulkeutuu nyt. Jos mahdollista, suorita päivitys käyttämättä ssh-"
"yhteyttä."

#: ../DistUpgrade/DistUpgradeController.py:251
msgid "Continue running under SSH?"
msgstr "Jatka käyttäen SSH:ta?"

#: ../DistUpgrade/DistUpgradeController.py:252
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Nykyinen istunto on muodostettu ssh-protokollan välityksellä. Päivitys ssh:n "
"välityksellä ei ole suositeltavaa, sillä mahdollisessa virhetilanteessa "
"palautuminen saattaa olla vaikeaa.\n"
"\n"
"Jos jatkat, ylimääräinen ssh-palvelu avataan porttiin '%s'.\n"
"Haluatko jatkaa?"

#: ../DistUpgrade/DistUpgradeController.py:267
msgid "Starting additional sshd"
msgstr "Käynnistän ylimääräisen SSH-palvelimen"

#: ../DistUpgrade/DistUpgradeController.py:268
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Jotta toipuminen ongelmatilanteesta olisi helpompaa,  käynnistetään porttiin "
"\"%s\" ylimääräinen SSH-palvelin. Jos jokin menee pieleen nykyisen SSH-"
"istunnon kanssa, voit silti yhdistää tähän uuteen.\n"

#: ../DistUpgrade/DistUpgradeController.py:339
#: ../DistUpgrade/DistUpgradeController.py:384
msgid "Can not upgrade"
msgstr "Päivitys ei onnistu"

#: ../DistUpgrade/DistUpgradeController.py:340
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Tämä työkalu ei tue päivitystä \"%s\" -> \"%s\"."

#: ../DistUpgrade/DistUpgradeController.py:349
msgid "Sandbox setup failed"
msgstr "Hiekkalaatikon asettaminen epäonnistui"

#: ../DistUpgrade/DistUpgradeController.py:350
msgid "It was not possible to create the sandbox environment."
msgstr "Hiekkalaatikkoympäristöä ei ollut mahdollista luoda."

#: ../DistUpgrade/DistUpgradeController.py:356
msgid "Sandbox mode"
msgstr "Hiekkalaatikko-tila"

#: ../DistUpgrade/DistUpgradeController.py:357
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"Tämä päivitys suoritetaan hiekkalaatikko- eli testitilassa. Kaikki "
"kohteeseen ”%s” kirjoitetut muutokset hukataan, kun seuraavan kerran "
"käynnistetään uudelleen."

#: ../DistUpgrade/DistUpgradeController.py:385
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Python-asennuksesi on viallinen. Korjaa symbolinen linkki "
"\"/usr/bin/python\"."

#: ../DistUpgrade/DistUpgradeController.py:411
msgid "Package 'debsig-verify' is installed"
msgstr "Paketti \"debsig-verify\" on asennettu"

#: ../DistUpgrade/DistUpgradeController.py:412
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Päivitys ei voi jatkua kyseisen paketin ollessa asennettu.\n"
"Poista paketti Synaptic-ohjelmalla tai komennolla \"apt-get remove debsig-"
"verify\" ja käynnistä päivitys uudelleen."

#: ../DistUpgrade/DistUpgradeController.py:422
msgid "Include latest updates from the Internet?"
msgstr "Sisällytä uusimmat päivitykset Internetistä?"

#: ../DistUpgrade/DistUpgradeController.py:423
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Päivitysjärjestelmä voi ladata Internetistä uusimmat päivitykset "
"automaattisesti ja asentaa ne päivityksen aikana. Jos sinulla on "
"verkkoyhteys käytettävissä, tämä on hyvin suositeltavaa.\n"
"\n"
"Päivitys kestää tällöin pidempään, mutta valmistuttuaan järjestelmä on "
"täysin ajan tasalla. Jos et tee tätä nyt, uusimmat päivitykset tulee joka "
"tapauksessa ladata pian päivityksen jälkeen.\n"
"Jos vastaat \"ei\" tähän, verkkoa ei käytetä lainkaan."

#: ../DistUpgrade/DistUpgradeController.py:647
#, python-format
msgid "disabled on upgrade to %s"
msgstr "otettu pois käytöstä päivitettäessä versioon %s"

#: ../DistUpgrade/DistUpgradeController.py:674
msgid "No valid mirror found"
msgstr "Sopivaa peilipalvelinta ei löytynyt"

#: ../DistUpgrade/DistUpgradeController.py:675
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Ohjelmalähteen tietoja luettaessa ei löytynyt merkintää päivityksestä. Tämä "
"voi tapahtua jos käytössäsi on yksityinen ohjelmalähde tai ohjelmalähteen "
"tiedot eivät ole ajan tasalla.\n"
"\n"
"Korvataanko \"sources.list\"-tiedosto joka tapauksessa? Jos valitset "
"\"Kyllä\", jokainen \"%s\" muutetaan muotoon \"%s\"."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:695
msgid "Generate default sources?"
msgstr "Lisätäänkö oletusohjelmalähteet?"

#: ../DistUpgrade/DistUpgradeController.py:696
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"”sources.list”-tiedostosta ei löytynyt kelvollista riviä lähteelle ”%s”.\n"
"\n"
"Tulisiko lähteen ”%s” oletusrivit lisätä? Jos valitse ”Ei”, päivitys "
"keskeytyy."

#: ../DistUpgrade/DistUpgradeController.py:731
msgid "Repository information invalid"
msgstr "Virhe ohjelmavarastotiedoissa"

#: ../DistUpgrade/DistUpgradeController.py:732
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug."
msgstr ""
"Ohjelmavarastotietojen päivittäminen johti epäkelpoon tiedostoon. Tee "
"asiasta virheraportti."

#: ../DistUpgrade/DistUpgradeController.py:738
msgid "Third party sources disabled"
msgstr "Kolmannen osapuolen ohjelmalähteet poissa käytöstä"

#: ../DistUpgrade/DistUpgradeController.py:739
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Jotkin kolmannen osapuolen lähteet sources.list-tiedostossa ovat nyt poissa "
"käytöstä. Voit ottaa ne uudelleen käyttöön päivityksen jälkeen "
"\"Ohjelmalähteet\"-työkalulla tai pakettienhallintaohjelmalla."

#: ../DistUpgrade/DistUpgradeController.py:779
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paketti ristiriitaisessa tilassa"
msgstr[1] "Paketteja ristiriitaisessa tilassa"

#: ../DistUpgrade/DistUpgradeController.py:782
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Paketti ”%s” on ristiriitaisessa tilassa ja se tulee asentaa uudelleen, "
"mutta sen pakettitiedostoa ei löydy. Asenna paketti uudelleen käsin tai "
"poista se järjestelmästä."
msgstr[1] ""
"Paketit ”%s” ovat ristiriitaisessa tilassa ja ne tulee asentaa uudelleen, "
"mutta niiden pakettitiedostoja ei löydy. Asenna paketit uudelleen käsin tai "
"poista ne järjestelmästä."

#: ../DistUpgrade/DistUpgradeController.py:830
msgid "Error during update"
msgstr "Virhe päivitettäessä"

#: ../DistUpgrade/DistUpgradeController.py:831
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Päivitettäessä tapahtui virhe. Tämä on yleensä jonkinlainen verkko-ongelma. "
"Tarkista verkkoyhteytesi toiminta ja yritä uudelleen."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:840
#: ../UpdateManager/UpdateManager.py:573
msgid "Not enough free disk space"
msgstr "Levytilaa ei ole riittävästi"

#: ../DistUpgrade/DistUpgradeController.py:841
#, python-format
msgid ""
"The upgrade is now aborted. The upgrade needs a total of %s free space on "
"disk '%s'. Please free at least an additional %s of disk space on '%s'. "
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Päivitys keskeytyy. Päivittäminen vaatii vähintään %s vapaata tilaa levyllä "
"”%s”. Vapauta lisää tilaa vähintään %s levyllä ”%s”. Tyhjennä roskakori ja "
"poista aiempien asennusten väliaikaiset pakettitiedostot käyttämällä "
"päätteessä komentoa ”sudo apt-get clean”."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:869
#: ../DistUpgrade/DistUpgradeController.py:1607
msgid "Calculating the changes"
msgstr "Lasketaan muutoksia"

#: ../DistUpgrade/DistUpgradeController.py:885
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community."
msgstr ""
"Canonical Ltd. ei enää tue seuraavia ohjelmapaketteja. Niille voi edelleen "
"saada tukea yhteisöltä."

#: ../DistUpgrade/DistUpgradeController.py:892
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community.\n"
"\n"
"These packages will be suggested for removal at the end of the upgrade "
"because you have not enabled the community maintained software channel "
"(universe)."
msgstr ""
"Canonical Ltd. ei enää tue seuraavia ohjelmapaketteja. Niille voi edelleen "
"saada tukea yhteisöltä.\n"
"\n"
"Näitä paketteja ehdotetaan poistettavaksi päivityksen päätteeksi, koska et "
"ole ottanut käyttöön yhteisön ylläpitämien ohjelmistojen kanavaa "
"(”universe”)."

#: ../DistUpgrade/DistUpgradeController.py:900
msgid "Support for some applications ended"
msgstr "Tuki joillekin sovelluksille on loppunut"

#. ask the user if he wants to do the changes
#: ../DistUpgrade/DistUpgradeController.py:920
msgid "Do you want to start the upgrade?"
msgstr "Haluatko aloittaa päivityksen?"

#: ../DistUpgrade/DistUpgradeController.py:985
msgid "Upgrade canceled"
msgstr "Päivitys peruttiin"

#: ../DistUpgrade/DistUpgradeController.py:986
msgid ""
"The upgrade cancels now and the original system state is restored. You can "
"resume the upgrade at a later time."
msgstr ""
"Päivitys perutaan ja järjestelmä palautetaan alkuperäiseen tilaan. Voit "
"palata päivitykseen myöhemmin."

#: ../DistUpgrade/DistUpgradeController.py:991
#: ../DistUpgrade/DistUpgradeController.py:1094
msgid "Could not download the upgrades"
msgstr "Päivityksiä ei voitu noutaa"

#: ../DistUpgrade/DistUpgradeController.py:992
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far are kept."
msgstr ""
"Päivitys keskeytyy. Tarkista Internet-yhteytesi tai asennuslähteesi (esim. "
"CD-levy) toiminta ja yritä uudelleen. Kaikki toistaiseksi noudetut tiedostot "
"säilytetään."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1049
#: ../DistUpgrade/DistUpgradeController.py:1082
#: ../DistUpgrade/DistUpgradeController.py:1187
msgid "Error during commit"
msgstr "Virhe suoritettaessa"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1051
#: ../DistUpgrade/DistUpgradeController.py:1084
#: ../DistUpgrade/DistUpgradeController.py:1226
msgid "Restoring original system state"
msgstr "Palautetaan alkuperäistä järjestelmän tilaa"

#: ../DistUpgrade/DistUpgradeController.py:1052
#: ../DistUpgrade/DistUpgradeController.py:1066
#: ../DistUpgrade/DistUpgradeController.py:1085
msgid "Could not install the upgrades"
msgstr "Päivityksiä ei voitu asentaa"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1057
msgid ""
"The upgrade is now aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Päivitys keskeytyy. Järjestelmäsi voi olla toimimattomassa tilassa. "
"Korjaustoimenpiteet suoritetaan nyt (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1062
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Tee tästä virheraportti paketille ”update-manager” ja sisällytä raporttiin "
"tiedostot hakemistosta /var/log/dist-upgrade/.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1095
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Päivitys keskeytyy. Tarkista Internet-yhteytesi tai asennuslähteesi (esim. "
"CD-levy) toiminta ja yritä uudelleen. "

#: ../DistUpgrade/DistUpgradeController.py:1175
msgid "Remove obsolete packages?"
msgstr "Poistetaanko vanhentuneet paketit?"

#: ../DistUpgrade/DistUpgradeController.py:1176
#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Keep"
msgstr "_Säilytä"

#: ../DistUpgrade/DistUpgradeController.py:1176
msgid "_Remove"
msgstr "_Poista"

#: ../DistUpgrade/DistUpgradeController.py:1188
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Siistimisvaiheessa ilmeni ongelma. Lisätietoja alla olevassa viestissä. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1264
msgid "Required depends is not installed"
msgstr "Vaadittuja riippuvuuksia ei ole asennettu"

#: ../DistUpgrade/DistUpgradeController.py:1265
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Vaadittu riippuvuus \"%s\" ei ole asennettu. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1510
#: ../DistUpgrade/DistUpgradeController.py:1577
msgid "Checking package manager"
msgstr "Tarkistetaan pakettienhallintaa"

#: ../DistUpgrade/DistUpgradeController.py:1515
msgid "Preparing the upgrade failed"
msgstr "Päivityksen valmistelu epäonnistui"

#: ../DistUpgrade/DistUpgradeController.py:1516
msgid ""
"Preparing the system for the upgrade failed. Please report this as a bug "
"against the 'update-manager' package and include the files in /var/log/dist-"
"upgrade/ in the bug report."
msgstr ""
"Järjestelmän valmistelu päivitystä varten epäonnistui. Tee tästä "
"virheraportti paketille ”update-manager” ja liitä mukaan tiedostot "
"hakemistosta /var/log/dist-upgrade/."

#: ../DistUpgrade/DistUpgradeController.py:1532
msgid "Getting upgrade prerequisites failed"
msgstr "Päivityksen esivaatimusten nouto epäonnistui"

#: ../DistUpgrade/DistUpgradeController.py:1533
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug against the 'update-manager' package and include "
"the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"Järjestelmä ei onnistunut noutamaan päivityksen esivaatimuksia. Päivitys "
"keskeytyy nyt ja järjestelmän alkuperäinen tila palautuu.\n"
"\n"
"Ilmoita tästä virheraportilla paketille ”update-manager” ja sisällytä "
"raporttiin tiedostot hakemistosta /var/log/dist-upgrade/."

#: ../DistUpgrade/DistUpgradeController.py:1563
msgid "Updating repository information"
msgstr "Päivitetään ohjelmavarastotietoja"

#: ../DistUpgrade/DistUpgradeController.py:1595
msgid "Invalid package information"
msgstr "Pakettitiedot viallisia"

#: ../DistUpgrade/DistUpgradeController.py:1596
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"Pakettitietojen päivitysten jälkeen pakollista ohjelmapakettia ”%s” ei enää "
"löydetty.\n"
"Tämä merkitsee vakavaa virhetilannetta, tee tästä virheraportti paketille "
"”update-manager” ja sisällytä tiedostot hakemistosta /var/log/dist-upgrade/ "
"raporttiin."

#: ../DistUpgrade/DistUpgradeController.py:1613
#: ../DistUpgrade/DistUpgradeController.py:1661
msgid "Fetching"
msgstr "Noudetaan"

#: ../DistUpgrade/DistUpgradeController.py:1619
#: ../DistUpgrade/DistUpgradeController.py:1665
msgid "Upgrading"
msgstr "Päivitetään"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1624
#: ../DistUpgrade/DistUpgradeController.py:1667
#: ../DistUpgrade/DistUpgradeController.py:1674
#: ../DistUpgrade/DistUpgradeController.py:1685
msgid "Upgrade complete"
msgstr "Päivitys on valmis"

#: ../DistUpgrade/DistUpgradeController.py:1625
#: ../DistUpgrade/DistUpgradeController.py:1668
#: ../DistUpgrade/DistUpgradeController.py:1675
msgid ""
"The upgrade is completed but there were errors during the upgrade process."
msgstr "Päivitys on valmis mutta päivityksen aikana tapahtui virheitä."

#: ../DistUpgrade/DistUpgradeController.py:1632
msgid "Searching for obsolete software"
msgstr "Etsitään vanhentuneita ohjelmistoja"

#: ../DistUpgrade/DistUpgradeController.py:1637
msgid "System upgrade is complete."
msgstr "Järjestelmän päivitys on valmis."

#: ../DistUpgrade/DistUpgradeController.py:1686
msgid "The partial upgrade was completed."
msgstr "Osittainen päivitys valmistui."

#: ../DistUpgrade/DistUpgradeQuirks.py:149
msgid "No ARMv6 CPU"
msgstr "Ei ARMv6-suoritinta"

#: ../DistUpgrade/DistUpgradeQuirks.py:150
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Järjestelmässäsi on ARM-suoritin jonka arkkitehtuuri on vanhempi kuin ARMv6. "
"Kaikki Karmicin ohjelmapaketit on optimoitu siten, että ne vaativat "
"vähintään ARMv6-arkkitehtuuria tukevan suorittimen. Tästä syystä "
"järjestelmäsi päivittäminen uuteen Ubuntun julkaisuun ei ole mahdollista."

#: ../DistUpgrade/DistUpgradeQuirks.py:164
msgid "No init available"
msgstr "Init-palvelua ei löydy"

#: ../DistUpgrade/DistUpgradeQuirks.py:165
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 9.10 cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Järjestelmä vaikuttaa olevan virtuaalisoitu ympäristö jossa ei ole Init-"
"palvelua, esimerkiksi Linux-VServer. Ubuntu 9.10 ei toimi tällaisessa "
"järjestelmässä, joten virtuaalikoneen asetuksia on muutettava ennen Ubuntu "
"9.10:n käyttämistä.\n"
"\n"
"Haluatko varmasti jatkaa päivitystä?"

#. TRANSLATORS: this string is not displayed in the 9.10 upgrade
#. (but it will be used for 8.04 -> 10.04 LTS upgrades)
#: ../DistUpgrade/DistUpgradeQuirks.py:186
#: ../DistUpgrade/DistUpgradeQuirks.py:268
#: ../DistUpgrade/DistUpgradeQuirks.py:295
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Päivitys saattaa poistaa käytöstä visuaalisia tehosteita ja huonontaa "
"joidenkin pelien ja graafisesti raskaiden ohjelmien suorituskykyä."

#: ../DistUpgrade/DistUpgradeQuirks.py:190
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Järjestelmässäsi on käytössä AMD:n “fglrx“-näytönohjainajuri. Ubuntu "
"10.04:ssä ei ole tästä ajurista versiota, joka toimii näytönohjaimesi "
"kanssa. \n"
"\n"
"Jatketaanko päivitystä?"

#: ../DistUpgrade/DistUpgradeQuirks.py:272
#: ../DistUpgrade/DistUpgradeQuirks.py:299
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Järjestelmässäsi on käytössä NVIDIAn “nvidia“-näytönohjainajuri. Ubuntu "
"10.04:ssä ei ole tästä ajurista versiota, joka toimii näytönohjaimesi "
"kanssa. \n"
"\n"
"Jatketaanko päivitystä?"

#: ../DistUpgrade/DistUpgradeQuirks.py:473
msgid "evms in use"
msgstr "evms on käytössä"

#: ../DistUpgrade/DistUpgradeQuirks.py:474
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"/proc/mounts -tiedoston mukaan järjestelmässäsi on käytössä evms-"
"taltionhallinta. evms-ohjelmistoa ei enää tueta, joten ole hyvä ja poista se "
"käytöstä ja suorita päivitys uudelleen."

#: ../DistUpgrade/DistUpgradeMain.py:43
msgid "Sandbox upgrade using aufs"
msgstr "Hiekkalaatikkopäivitys aufs:ää käyttäen"

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Käytä annettua polkua päivitettyjen pakettien etsimiseksi CD-levyltä."

#: ../DistUpgrade/DistUpgradeMain.py:51
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Valitse käyttöliittymä. Tällä hetkellä valittavissa ovat: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:54
msgid "*DEPRECATED* this option will be ignore"
msgstr "*VANHENTUNUT* tämä valinta jätetään huomiotta"

#: ../DistUpgrade/DistUpgradeMain.py:57
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Suorita vain osittainen päivitys (ei sources.list-uudelleenkirjoitusta)"

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Set datadir"
msgstr "Aseta datahakemisto"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:109
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Aseta \"%s\" asemaan \"%s\""

#: ../DistUpgrade/DistUpgradeViewGtk.py:129
#: ../DistUpgrade/DistUpgradeViewKDE.py:204
msgid "Fetching is complete"
msgstr "Nouto on valmis"

#: ../DistUpgrade/DistUpgradeViewGtk.py:141
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Noudetaan tiedostoa %li/%li nopeudella %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:142
#: ../DistUpgrade/DistUpgradeViewGtk.py:277
#: ../DistUpgrade/DistUpgradeViewKDE.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:358
#, python-format
msgid "About %s remaining"
msgstr "Noin %s jäljellä"

#: ../DistUpgrade/DistUpgradeViewGtk.py:144
#: ../DistUpgrade/DistUpgradeViewKDE.py:220
#, python-format
msgid "Fetching file %li of %li"
msgstr "Noudetaan tiedostoa %li/%li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:176
#: ../DistUpgrade/DistUpgradeViewKDE.py:254
msgid "Applying changes"
msgstr "Muutoksia toteutetaan"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:201
#: ../DistUpgrade/DistUpgradeViewKDE.py:267
msgid "dependency problems - leaving unconfigured"
msgstr "riippuvuusongelmia - jätetään asetukset säätämättä"

#: ../DistUpgrade/DistUpgradeViewGtk.py:206
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid "Could not install '%s'"
msgstr "Pakettia \"%s\" ei voitu asentaa"

#: ../DistUpgrade/DistUpgradeViewGtk.py:207
#: ../DistUpgrade/DistUpgradeViewKDE.py:270
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Päivitys jatkuu mutta paketti %s ei välttämättä toimi päivityksen jälkeen. "
"Harkitse vikailmoituksen tekemistä virheestä."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:223
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Korvataanko alkuperäisestä muutettu asetustiedosto\n"
"\"%s\"?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:292
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Tähän asetustiedostoon tehdyt muutokset menetetään, jos se korvataan "
"uudemmalla versiolla."

#: ../DistUpgrade/DistUpgradeViewGtk.py:237
#: ../DistUpgrade/DistUpgradeViewKDE.py:310
msgid "The 'diff' command was not found"
msgstr "Komentoa 'diff' ei löytynyt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:442
#: ../DistUpgrade/DistUpgradeViewText.py:88
msgid "A fatal error occurred"
msgstr "Tapahtui vakava virhe"

#: ../DistUpgrade/DistUpgradeViewGtk.py:443
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Ilmoita tästä virheraportilla (jos et ole niin jo tehnyt) ja sisällytä "
"siihen tiedostot /var/log/dist-upgrade/main.log ja /var/log/dist-"
"upgrade/apt.log. Päivitys keskeytyy.\n"
"Alkuperäinen sources.list tallennettiin nimellä "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:460
msgid "Ctrl-c pressed"
msgstr "Ctrl-c painettu"

#: ../DistUpgrade/DistUpgradeViewGtk.py:461
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Tämä keskeyttää toiminnon ja saattaa jättää järjestelmän rikkinäiseen "
"tilaan. Haluatko varmasti tehdä sen?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:606
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Jottei tietoja häviäisi, sulje kaikki avoinna olevat ohjelmat ja asiakirjat."

#: ../DistUpgrade/DistUpgradeViewGtk.py:620
#, python-format
msgid "<b>Downgrade %s</b>"
msgstr "<b>Vaihda %s vanhempaan versioon</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:622
#: ../DistUpgrade/DistUpgradeViewKDE.py:758
#, python-format
msgid "Remove %s"
msgstr "Poista %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:627
#: ../DistUpgrade/DistUpgradeViewKDE.py:760
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Poistetaan (oli automaattisesti asennettu) %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:630
#: ../DistUpgrade/DistUpgradeViewKDE.py:762
#, python-format
msgid "Install %s"
msgstr "Asenna %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:632
#: ../DistUpgrade/DistUpgradeViewKDE.py:764
#, python-format
msgid "Upgrade %s"
msgstr "Päivitä %s"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:191
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Media Change"
msgstr "Median vaihto"

#: ../DistUpgrade/DistUpgradeViewKDE.py:322
msgid "Show Difference >>>"
msgstr "Näytä erot >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:325
msgid "<<< Hide Difference"
msgstr "<<< Piilota erot"

#: ../DistUpgrade/DistUpgradeViewKDE.py:539
msgid "Error"
msgstr "Virhe"

#: ../DistUpgrade/DistUpgradeViewKDE.py:551
msgid "&Cancel"
msgstr "&Peru"

#: ../DistUpgrade/DistUpgradeViewKDE.py:553
#: ../DistUpgrade/DistUpgradeViewKDE.py:792
msgid "&Close"
msgstr "&Sulje"

#: ../DistUpgrade/DistUpgradeViewKDE.py:599
msgid "Show Terminal >>>"
msgstr "Näytä pääte >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:602
msgid "<<< Hide Terminal"
msgstr "<<< Piilota pääte"

#: ../DistUpgrade/DistUpgradeViewKDE.py:682
msgid "Information"
msgstr "Tiedot"

#: ../DistUpgrade/DistUpgradeViewKDE.py:732
#: ../DistUpgrade/DistUpgradeViewKDE.py:775
#: ../DistUpgrade/DistUpgradeViewKDE.py:778 ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Details"
msgstr "Yksityiskohdat"

#: ../DistUpgrade/DistUpgradeViewKDE.py:788
#: ../DistUpgrade/DistUpgradeViewText.py:206
msgid "Restart required"
msgstr "Uudelleenkäynnistys vaaditaan"

#: ../DistUpgrade/DistUpgradeViewKDE.py:788
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr ""
"<b><big>Käynnistä järjestelmä uudelleen päivityksen "
"viimeistelemiseksi</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:791 ../DistUpgrade/DistUpgrade.ui.h:25
#: ../data/glade/UpdateManager.ui.h:31
msgid "_Restart Now"
msgstr "_Uudelleenkäynnistä nyt"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:803
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Peru käynnissä oleva päivitys?</big></b>\n"
"\n"
"Järjestelmä voi olla käyttökelvottomassa tilassa, jos perut päivityksen. "
"Päivityksen jatkaminen on erittäin suositeltavaa."

#: ../DistUpgrade/DistUpgradeViewKDE.py:807
msgid "Cancel Upgrade?"
msgstr "Peru päivitys?"

#: ../DistUpgrade/DistUpgradeView.py:60
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li päivä"
msgstr[1] "%li päivää"

#: ../DistUpgrade/DistUpgradeView.py:62
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li tunti"
msgstr[1] "%li tuntia"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuutti"
msgstr[1] "%li minuuttia"

#: ../DistUpgrade/DistUpgradeView.py:65
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekunti"
msgstr[1] "%li sekuntia"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:78
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:93
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:128
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Nouto kestää noin %s 1Mbit-DSL-yhteydellä ja noin %s 56kbit-modeemilla."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:132
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Nouto kestää yhteydelläsi noin %s. "

#: ../DistUpgrade/DistUpgradeView.py:234 ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Preparing to upgrade"
msgstr "Valmistellaan päivitystä"

#: ../DistUpgrade/DistUpgradeView.py:235
msgid "Getting new software channels"
msgstr "Noudetaan uusia ohjelmalähteitä"

#: ../DistUpgrade/DistUpgradeView.py:236 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Getting new packages"
msgstr "Haetaan uusia paketteja"

#: ../DistUpgrade/DistUpgradeView.py:237 ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Installing the upgrades"
msgstr "Asennetaan päivityksiä"

#: ../DistUpgrade/DistUpgradeView.py:238 ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Cleaning up"
msgstr "Siistitään"

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:322
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d paketti tullaan poistamaan."
msgstr[1] "%d pakettia tullaan poistamaan."

#: ../DistUpgrade/DistUpgradeView.py:327
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d uusi paketti asennetaan."
msgstr[1] "%d uutta pakettia asennetaan."

#: ../DistUpgrade/DistUpgradeView.py:333
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d paketti päivitetään."
msgstr[1] "%d pakettia päivitetään."

#: ../DistUpgrade/DistUpgradeView.py:338
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Noudettavaa yhteensä %s. "

#: ../DistUpgrade/DistUpgradeView.py:342
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Ubuntu-päivityksen noutaminen ja asentaminen saattaa kestää useita tunteja. "
"Noutamisen päätyttyä päivityksen suoritusta ei voi enää keskeyttää."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:348 ../UpdateManager/UpdateManager.py:503
msgid "Your system is up-to-date"
msgstr "Järjestelmäsi on ajan tasalla"

#: ../DistUpgrade/DistUpgradeView.py:349
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "Päivityksiä ei ole saatavilla järjestelmälle. Päivitys keskeytyy."

#: ../DistUpgrade/DistUpgradeView.py:362
msgid "Reboot required"
msgstr "Uudelleenkäynnistys vaaditaan"

#: ../DistUpgrade/DistUpgradeView.py:363
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Päivitys on valmis ja uudelleenkäynnistys vaaditaan. Haluatko käynnistää "
"tietokoneen uudelleen nyt?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../DistUpgrade/DistUpgradeFetcherCore.py:136
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:136
msgid "Could not run the upgrade tool"
msgstr "Päivitystyökalua ei voitu suorittaa"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:136
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:136
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug"
msgstr ""
"Tämä on luultavasti ohjelmavirhe päivitystyökalussa. Ilmoita tästä "
"virheraportilla."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:205
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:205
msgid "Upgrade tool signature"
msgstr "Päivitystyökalun allekirjoitus"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:208
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:208
msgid "Upgrade tool"
msgstr "Päivitystyökalu"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:237
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:237
msgid "Failed to fetch"
msgstr "Nouto epäonnistui"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:238
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Päivityksen noutaminen epäonnistui. Tämä voi johtua verkko-ongelmasta. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:242
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:242
msgid "Failed to extract"
msgstr "Purku epäonnistui"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:243
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Päivityksen purkaminen epäonnistui. Tämä voi johtua ongelmasta "
"verkkoyhteydessä tai palvelimessa. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Verification failed"
msgstr "Varmennus epäonnistui"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Päivityksen tarkistaminen epäonnistui. Tämä voi johtua ongelmasta "
"verkkoyhteydessä tai palvelimessa. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid "Authentication failed"
msgstr "Todennus epäonnistui"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:255
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Päivityksen todennus epäonnistui. Tämä voi johtua ongelmasta "
"verkkoyhteydessä tai palvelimessa. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../DistUpgrade/DistUpgradeFetcherCore.py:275
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:275
msgid "Can not run the upgrade"
msgstr "Päivitystä ei voi suorittaa"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:270
msgid ""
"This usually is caused by a system were /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Tämä johtuu yleensä järjestelmästä, jossa /tmp on liitetty valitsimella "
"\"noexec\". Liitä uudelleen ilman \"noexec\"-valitsinta ja suorita päivitys "
"uudelleen."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:276
#, python-format
msgid "The error message is '%s'."
msgstr "Virheviesti oli \"%s\"."

#: ../DistUpgrade/DistUpgradeViewText.py:89
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Ilmoita tästä virheraportilla ja sisällytä siihen tiedostot /var/log/dist-"
"upgrade/main.log ja /var/log/dist-upgrade/apt.log. Päivitys keskeytyy nyt.\n"
"Alkuperäinen sources.list tallennettiin nimellä "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:113
msgid "Aborting"
msgstr "Peruutetaan"

#: ../DistUpgrade/DistUpgradeViewText.py:118
msgid "Demoted:\n"
msgstr "Alennettu:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:150
#: ../DistUpgrade/DistUpgradeViewText.py:172
#: ../DistUpgrade/DistUpgradeViewText.py:179
msgid "Continue [yN] "
msgstr "Jatka [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:150
#: ../DistUpgrade/DistUpgradeViewText.py:172
msgid "Details [d]"
msgstr "Yksityiskohdat [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:154
#: ../DistUpgrade/DistUpgradeViewText.py:182
msgid "y"
msgstr "y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:189
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:160
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:164
#, python-format
msgid "Remove: %s\n"
msgstr "Poista: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:167
#, python-format
msgid "Install: %s\n"
msgstr "Asenna: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:170
#, python-format
msgid "Upgrade: %s\n"
msgstr "Päivitä: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:186
msgid "Continue [Yn] "
msgstr "Jatka [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:207
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Uudelleenkäynnistys vaaditaan päivityksen saattamiseksi loppuun.\n"
"Jos valitset \"y\" (\"yes\" eli \"kyllä\") järjestelmä käynnistetään "
"uudelleen."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Peruuta käynnissä oleva päivitys?</big></b>\n"
"\n"
"Järjestelmä voi olla käyttökelvottomassa tilassa, jos peruutat päivityksen. "
"Päivityksen jatkaminen on erittäin suositeltavaa."

#: ../DistUpgrade/DistUpgrade.ui.h:5
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Käynnistä järjestelmä uudelleen, jotta päivitys valmistuu</big></b>\n"
"\n"
"Tallenna työsi ennen jatkamista."

#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Aloitetaanko päivitys?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "<b><big>Upgrading Ubuntu to version 10.04 LTS</big></b>"
msgstr "<b><big>Päivitetään Ubuntu versioon 10.04 LTS</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Difference between the files"
msgstr "Tiedostojen väliset erot"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Distribution Upgrade"
msgstr "Jakelupäivitys"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Restarting the computer"
msgstr "Käynnistetään tietokone uudelleen"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Setting new software channels"
msgstr "Asetetaan uusia ohjelmalähteitä"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "Terminal"
msgstr "Pääte"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Cancel Upgrade"
msgstr "_Peru päivitys"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Continue"
msgstr "_Jatka"

#: ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Replace"
msgstr "_Korvaa"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Report Bug"
msgstr "_Tee virheraportti"

#: ../DistUpgrade/DistUpgrade.ui.h:26
msgid "_Resume Upgrade"
msgstr "_Jatka päivitystä"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "_Start Upgrade"
msgstr "_Aloita päivitys"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:42
msgid "Please wait, this can take some time."
msgstr "Odota, tämä voi kestää jonkin aikaa."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:44
msgid "Update is complete"
msgstr "Päivitys on valmis"

#: ../UpdateManager/DistUpgradeFetcher.py:85
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "Could not find the release notes"
msgstr "Julkaisutietoja ei löytynyt"

#: ../UpdateManager/DistUpgradeFetcher.py:86
#: ../UpdateManager/DistUpgradeFetcherKDE.py:104
msgid "The server may be overloaded. "
msgstr "Palvelin voi olla ylikuormitettu. "

#: ../UpdateManager/DistUpgradeFetcher.py:96
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Could not download the release notes"
msgstr "Julkaisutietoja ei voitu noutaa"

#: ../UpdateManager/DistUpgradeFetcher.py:97
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Please check your internet connection."
msgstr "Tarkista Internet-yhteytesi toimivuus."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:62
#: ../UpdateManager/DistUpgradeFetcherKDE.py:85
msgid "Upgrade"
msgstr "Päivitä"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:89
#: ../data/glade/UpdateManager.ui.h:12
#: ../data/glade/UpgradePromptDialog.ui.h:4
msgid "Release Notes"
msgstr "Julkaisutiedot"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:128
#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
msgid "Downloading additional package files..."
msgstr "Noudetaan tarvittavia pakettitiedostoja..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Tiedosto %s/%s nopeudella %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
#, python-format
msgid "File %s of %s"
msgstr "Tiedosto %s/%s"

#: ../UpdateManager/ChangelogViewer.py:70
msgid "Open Link in Browser"
msgstr "Avaa linkki selaimessa"

#: ../UpdateManager/ChangelogViewer.py:72
msgid "Copy Link to Clipboard"
msgstr "Kopioi linkki leikepöydälle"

#: ../UpdateManager/GtkProgress.py:116
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Noudetaan tiedostoa %(current)li/%(total)li nopeudella %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:121
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Noudetaan tiedostoa %(current)li/%(total)li"

#: ../UpdateManager/UpdateManager.py:304
#, python-format
msgid "Version %s: \n"
msgstr "Versio %s: \n"

#: ../UpdateManager/UpdateManager.py:361
msgid "Downloading list of changes..."
msgstr "Noudetaan muutosluetteloa..."

#: ../UpdateManager/UpdateManager.py:401
msgid "_Uncheck All"
msgstr "_Poista valinnat"

#: ../UpdateManager/UpdateManager.py:407
msgid "_Check All"
msgstr "_Valitse kaikki"

#: ../UpdateManager/UpdateManager.py:451
#, python-format
msgid ""
"Download size: %s\n"
"%s selected."
msgstr ""
"Noudettavan koko: %s\n"
"%s valittu."

#: ../UpdateManager/UpdateManager.py:456
msgid "Unknown download size"
msgstr "Latauksen kokoa ei tiedetä"

#: ../UpdateManager/UpdateManager.py:464
msgid ""
"It is unknown when the package information was updated last. Please try "
"clicking on the 'Check' button to update the information."
msgstr ""
"Pakettitietojen viimeisintä päivityskertaa ei ole tiedossa. Kokeile "
"napsauttaa \"Tarkista\"-painiketta päivittääksesi tiedot."

#: ../UpdateManager/UpdateManager.py:473
#, python-format
msgid "The package information was last updated %s day ago."
msgid_plural "The package information was last updated %s days ago."
msgstr[0] "Pakettitiedot päivitetty viimeksi %s päivä sitten."
msgstr[1] "Pakettitiedot päivitetty viimeksi %s päivää sitten."

#: ../UpdateManager/UpdateManager.py:477
#, python-format
msgid "The package information was last updated %s hour ago."
msgid_plural "The package information was last updated %s hours ago."
msgstr[0] "Pakettitiedot päivitetty viimeksi %s tunti sitten."
msgstr[1] "Pakettitiedot päivitetty viimeksi %s tuntia sitten."

#: ../UpdateManager/UpdateManager.py:481
msgid "The package information was last updated less than one hour ago."
msgstr "Pakettitiedot päivitetty alle tunti sitten."

#: ../UpdateManager/UpdateManager.py:501 ../data/glade/UpdateManager.ui.h:23
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Ohjelmapäivitykset korjaavat ohjelmavirheitä ja mahdollisia turva-aukkoja "
"sekä tarjoavat uusia ominaisuuksia."

#: ../UpdateManager/UpdateManager.py:520
msgid "Welcome to Ubuntu"
msgstr "Tervetuloa Ubuntuun"

#: ../UpdateManager/UpdateManager.py:521
msgid ""
"These software updates have been issued since Ubuntu was released. If you "
"don't want to install them now, choose \"Update Manager\" from the "
"Administration Menu later."
msgstr ""
"Nämä ohjelmapäivitykset on tuotu tarjolle Ubuntun julkaisun jälkeen. Jos et "
"halua asentaa niitä nyt, valitse Järjestelmä-valikosta ”Päivitysten "
"hallinta” myöhemmin."

#: ../UpdateManager/UpdateManager.py:524
msgid "Software updates are available for this computer"
msgstr "Ohjelmapäivityksiä on saatavilla tälle tietokoneelle"

#: ../UpdateManager/UpdateManager.py:525
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Jos et halua asentaa niitä nyt, valitse Järjestelmä-valikosta ”Päivitysten "
"hallinta” myöhemmin."

#: ../UpdateManager/UpdateManager.py:574
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Päivitys tarvitsee yhteensä %s vapaata tilaa levyllä \"%s\". Vapauta "
"vähintään %s lisää levytilaa levyllä \"%s\". Tyhjennä roskakori, ja poista "
"väliaikaiset aiempien asennuksien paketit käyttämällä komentoa \"sudo apt-"
"get clean\"."

#: ../UpdateManager/UpdateManager.py:631
#: ../UpdateManagerHildon/UpdateManagerHildon.py:114
msgid "Reading package information"
msgstr "Luetaan pakettitietoja"

#: ../UpdateManager/UpdateManager.py:744
msgid "Could not initialize the package information"
msgstr "Pakettitietoja ei voitu alustaa"

#: ../UpdateManager/UpdateManager.py:745
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Pakettitietoja alustettaessa tapahtui virhe jota ei voitu korjata.\n"
"\n"
"Ilmoita tästä virheraportilla paketille \"update-manager\", ja sisällytä "
"raporttiin seuraava virheviesti:\n"

#: ../UpdateManager/UpdateManager.py:767
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Määriteltäessä päivitystä tapahtui virhe jota ei voitu korjata.\n"
"\n"
"Ilmoita tästä virheraportilla paketille \"update-manager\" ja sisällytä "
"raporttiin seuraava virheviesti:"

#: ../UpdateManager/UpdateManager.py:790
msgid " (New install)"
msgstr " (uusi asennus)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:794
#, python-format
msgid "(Size: %s)"
msgstr "(Koko: %s)"

#: ../UpdateManager/UpdateManager.py:796
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Versiosta %(old_version)s versioon %(new_version)s"

#: ../UpdateManager/UpdateManager.py:800
#, python-format
msgid "Version %s"
msgstr "Versio %s"

#: ../UpdateManager/UpdateManager.py:813
msgid "Your Ubuntu release is not supported anymore"
msgstr "Ubuntu-julkaisuasi ei tueta enää"

#: ../UpdateManager/UpdateManager.py:814
msgid ""
"You will not get any further security fixes or critical updates. Please "
"Upgrade to a later version of Ubuntu Linux."
msgstr ""
"Tälle Ubuntun versiolle ei enää tule turvallisuuspäivityksiä tai kriittisiä "
"päivityksiä. Ole hyvä ja päivitä uudempaan versioon."

#: ../UpdateManager/UpdateManager.py:821
msgid "Upgrade information"
msgstr "Tietoja päivityksestä"

#. print "on_button_dist_upgrade_clicked"
#: ../UpdateManager/UpdateManager.py:841 ../check-new-release-gtk:87
msgid "Downloading the release upgrade tool"
msgstr "Noudetaan julkaisupäivitystyökalua"

#: ../UpdateManager/UpdateManager.py:848
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Uusi Ubuntu-julkaisu ”%s” on saatavilla</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:887
msgid "Software index is broken"
msgstr "Ohjelmaluettelo on rikkinäinen"

#: ../UpdateManager/UpdateManager.py:888
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Mitään ohjelmia ei voida asentaa tai poistaa. Korjaa ongelma käyttämällä "
"Synaptic-pakettienhallintaa tai komentoa \"sudo apt-get install -f\" "
"päätteessä."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:136
msgid "Downloading Package Information"
msgstr "Noudetaan pakettitietoja"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:137
msgid ""
"The repositories are being checked for new, removed, or updated software "
"packages"
msgstr ""
"Ohjelmalähteistä tarkistetaan nyt uusia, poistuneita ja päivittyneitä "
"ohjelmapaketteja."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:145
msgid "Downloading Package Updates"
msgstr "Noudetaan pakettipäivityksiä"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:146
msgid ""
"The selected package updates are being downloaded and installed on the system"
msgstr "Valittuja pakettipäivityksiä noudetaan ja asennetaan järjestelmään."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:152
msgid "run_synaptic called with unknown action"
msgstr "run_synaptic:ia kutsuttiin tuntemattomalla toiminnolla"

#: ../UpdateManagerText/UpdateManagerText.py:26
msgid "Cancel"
msgstr "Peru"

#: ../UpdateManagerText/UpdateManagerText.py:27
msgid "Install"
msgstr "Asenna"

#: ../UpdateManagerText/UpdateManagerText.py:45
msgid "Building Updates List"
msgstr "Rakennetaan päivitysluetteloa"

#: ../UpdateManagerText/UpdateManagerText.py:48
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Tavallista päivitystä ei voi laskea. Suorita: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Tämä voi johtua seuraavista asioista:\n"
" * Edellinen päivitys ei valmistunut\n"
" * Ongelmia joidenkin asennettujen ohjelmistojen kanssa\n"
" * Käytössä on epävirallisia ohjelmapaketteja, joita Ubuntu ei tarjoa\n"
" * Kyse on tavallisista muutoksista Ubuntun esijulkaisuversiossa"

#: ../UpdateManagerText/UpdateManagerText.py:117
msgid "Downloading changelog"
msgstr "Noudetaan muutoslokia"

#: ../UpdateManager/Core/MyCache.py:124
#, python-format
msgid "Other updates (%s)"
msgstr "Muita päivityksiä (%s)"

#: ../UpdateManager/Core/MyCache.py:246
#, python-format
msgid ""
"Changes for the versions:\n"
"%s\n"
"%s\n"
"\n"
msgstr ""
"Muutokset tähän versioon:\n"
"%s\n"
"%s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:248
msgid "This change is not coming from a source that supports changelogs."
msgstr "Päivityksen tarjoava lähde ei tarjoa muutoslokeja."

#: ../UpdateManager/Core/MyCache.py:256
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Muutosloki ei sisällä mitään oleellisia muutoksia.\n"
"\n"
"Käytä osoitetta http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"kunnes muutokset tulevat saataville, tai yritä myöhemmin uudelleen."

#: ../UpdateManager/Core/MyCache.py:261
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Muutosluettelo ei ole vielä saatavilla.\n"
"\n"
"Käytä osoitetta http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"kunnes muutokset tulevat saataville, tai yritä myöhemmin uudelleen."

#: ../UpdateManager/Core/MyCache.py:267
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Muutosluettelon nouto epäonnistui. \n"
"Tarkista Internet-yhteytesi toimivuus."

#: ../UpdateManager/Core/UpdateList.py:50
msgid "Failed to detect distribution"
msgstr "Jakelun tunnistus epäonnistui"

#: ../UpdateManager/Core/UpdateList.py:51
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Järjestelmän tyypin tarkistamista tehtäessä tapahtui virhe \"%s\"."

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Important security updates"
msgstr "Tärkeät turvallisuuspäivitykset"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Recommended updates"
msgstr "Suositellut päivitykset"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Proposed updates"
msgstr "Ehdotetut päivitykset"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Backports"
msgstr "Takaisinsovitukset"

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Distribution updates"
msgstr "Jakelupäivitykset"

#: ../UpdateManager/Core/UpdateList.py:71
msgid "Other updates"
msgstr "Muut päivitykset"

#: ../UpdateManager/Core/UpdateList.py:80
msgid "Previous selected"
msgstr "Edellinen valittu"

#: ../data/glade/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Päivitykset tulee tarkistaa käsin</big></b>\n"
"Järjestelmä ei tällä hetkellä seuraa päivityksiä automaattisesti. Voit "
"muuttaa näitä asetuksia <i>Ohjelmalähteet</i>-ylläpitotyökalun "
"<i>Päivitykset</i>-välilehdellä."

#: ../data/glade/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Kaikkia päivityksiä ei voida asentaa</b></big>"

#: ../data/glade/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Akkukäytössä</b></big>"

#: ../data/glade/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Käynnistetään päivitysten hallintaa</b></big>"

#: ../data/glade/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Muutokset"

#: ../data/glade/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "_Tarkista"

#: ../data/glade/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "_Jatka"

#: ../data/glade/UpdateManager.ui.h:10
msgid "Description"
msgstr "Kuvaus"

#: ../data/glade/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Päivityksen kuvaus"

#: ../data/glade/UpdateManager.ui.h:13
msgid "Restart _Later"
msgstr "Käynnistä uudelleen _myöhemmin"

#: ../data/glade/UpdateManager.ui.h:14
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Tee osittainen päivitys ja päivitä niin monta ohjelmapakettia kuin "
"mahdollista. \n"
"\n"
"Tähän tilanteeseen päädytään seuraavista syistä:\n"
" * Edellistä päivitystä ei suoritettu loppuun asti\n"
" * Jonkun asennetun ohjelman kanssa on ongelmia\n"
" * Järjestelmään on asennettu kolmannen osapuolen ohjelmapaketteja\n"
" * Ubuntun esiversion normaalit muutokset estävät päivityksen"

#: ../data/glade/UpdateManager.ui.h:21
msgid "Settings..."
msgstr "Asetukset..."

#: ../data/glade/UpdateManager.ui.h:22
#: ../data/glade/UpgradePromptDialog.ui.h:5
msgid "Show progress of individual files"
msgstr "Näytä tiedostojen edistyminen"

#: ../data/glade/UpdateManager.ui.h:24
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Tietokoneen tulee käynnistyä uudelleen päivityksien asennuksen "
"valmistumiseksi. Tallenna työsi ennen jatkamista."

#: ../data/glade/UpdateManager.ui.h:25
msgid "U_pgrade"
msgstr "_Päivitä"

#: ../data/glade/UpdateManager.ui.h:26 ../data/update-manager.desktop.in.h:3
#: ../data/update-manager-hildon.desktop.in.h:3
msgid "Update Manager"
msgstr "Päivitysten hallinta"

#: ../data/glade/UpdateManager.ui.h:27
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "Järjestelmä toimii akun varassa. Haluatko varmasti jatkaa?"

#: ../data/glade/UpdateManager.ui.h:28
msgid "_Hide this information in the future"
msgstr "P_iilota tämä tieto jatkossa"

#: ../data/glade/UpdateManager.ui.h:29
msgid "_Install Updates"
msgstr "_Asenna päivitykset"

#: ../data/glade/UpdateManager.ui.h:30
msgid "_Partial Upgrade"
msgstr "_Osittainen päivitys"

#: ../data/glade/UpdateManager.ui.h:32
msgid "_Upgrade"
msgstr "_Päivitä"

#: ../data/glade/UpdateManager.ui.h:33
msgid "updates"
msgstr "päivitykset"

#: ../data/glade/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Uusi Ubuntun versio on saatavilla. Haluatko aloittaa päivityksen?</b>"

#: ../data/glade/UpgradePromptDialog.ui.h:2
msgid "Ask Me Later"
msgstr "Kysy myöhemmin uudestaan"

#: ../data/glade/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Älä päivitä"

#: ../data/glade/UpgradePromptDialog.ui.h:6
msgid "Yes, Upgrade Now"
msgstr "Kyllä, päivitä nyt"

#: ../data/glade/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr ""
"Voit päivittää koska tahansa myöhemmin käynnistämällä Päivitysten hallinta -"
"sovelluksen ja napsauttamalla ”Päivitä”-painiketta."

#: ../data/glade/UpgradePromptDialog.ui.h:8
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Valitsin päivittämisen uuteen Ubuntun versioon"

#: ../data/update-manager.desktop.in.h:1
#: ../data/update-manager-hildon.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Näytä ja asenna saatavilla olevat päivitykset"

#: ../data/update-manager.desktop.in.h:2
#: ../data/update-manager-hildon.desktop.in.h:2
msgid "Software Updates"
msgstr "Ohjelmapäivitykset"

#: ../data/update-manager.schemas.in.h:1
msgid "*deprecated* Check for new distribution releases"
msgstr "*vanhentunut* Tarkista, onko jakelusta uusia julkaisuja"

#: ../data/update-manager.schemas.in.h:2
msgid "Auto close the install window after successful install"
msgstr "Sulje asennusikkuna automaattisesti onnistuneen asennuksen jälkeen"

#: ../data/update-manager.schemas.in.h:3
msgid "First run welcome message"
msgstr "Ensimmäisen käynnistyskerran tervetuloa-viesti"

#: ../data/update-manager.schemas.in.h:4
msgid ""
"If automatic checking for updates is disabled, you have to reload the "
"channel list manually. This option allows to hide the reminder shown in this "
"case."
msgstr ""
"Jos automaattinen päivitysten tarkistaminen on poissa käytöstä, täytyy "
"ohjelmaluettelot ladata käsin. Tämä valinta mahdollistaa tämän muistutuksen "
"piilottamisen."

#: ../data/update-manager.schemas.in.h:5
msgid "If this key is set a first run welcome message will be presented."
msgstr ""
"Jos tämä avain on asetettu, ensimmäisen käynnistyskerran tervetuloa-viesti "
"esitetään."

#: ../data/update-manager.schemas.in.h:6
msgid ""
"If this key is set the install window will be automatically closed on "
"successful installation."
msgstr ""
"Jos tämä avain on asetettu, asennusikkuna suljetaan automaattisesti "
"onnistuneen asennuksen tapahduttua."

#: ../data/update-manager.schemas.in.h:7
msgid ""
"If this key is set the main update list window will show version information "
"(from version to version)."
msgstr ""
"Jos tämä asetus on valittu, päivitysikkunassa näytetään, mikä versio "
"ohjelmasta asennetaan ja mikä versio on asennettuna."

#: ../data/update-manager.schemas.in.h:8
msgid "Remind to reload the channel list"
msgstr "Muistuta kanavaluettelon uudelleen lataamisesta"

#: ../data/update-manager.schemas.in.h:9
msgid "Show details of an update"
msgstr "Näytä päivityksen yksityiskohdat"

#: ../data/update-manager.schemas.in.h:10
msgid "Show version in update list"
msgstr "Näytä versiot päivityslistassa."

#: ../data/update-manager.schemas.in.h:11
msgid "Stores the size of the update-manager dialog"
msgstr "Tallentaa päivitystenhallintaikkunan koon"

#: ../data/update-manager.schemas.in.h:12
msgid ""
"Stores the state of the expander that contains the list of changes and the "
"description"
msgstr ""
"Tallentaa muutosluettelon ja kuvauksen näyttämän ikkunalaajennoksen tilan."

#: ../data/update-manager.schemas.in.h:13
msgid "The window size"
msgstr "Ikkunan koko"

#: ../data/update-manager.schemas.in.h:14
msgid ""
"This key is deprecated in favor of the file /etc/update-manager/release-"
"upgrades Check automatically if a new version of the current distribution is "
"available and offer to upgrade (if possible)."
msgstr ""
"Tämä avain on vanhentunut tiedoston /etc/update-manager/release-upgrades "
"tieltä. Tarkista automaattisesti, onko jakelusta saatavilla uudempaa "
"versiota ja tarjoa päivitystä (jos mahdollista)."

#: ../data/update-manager.schemas.in.h:15
msgid ""
"This will permanently hide the new release prompt from check-new-release-"
"gtk. Note that the small button in the main update-manager UI will still be "
"there."
msgstr ""
"Tämä piilottaa ilmoituksen uudesta julkaisusta pysyvästi check-new-release-"
"gtk:sta. Huomaa, että Päivitysten hallinnan käyttöliittymän pieni painike "
"pysyy edelleen näkyvillä."

#: ../data/update-manager.schemas.in.h:16
msgid "make check-new-release-gtk ignore a given new release"
msgstr ""
"muuta check-new-release-gtk jättämään huomiotta annettu uusi julkaisu"

#: ../update-manager:56 ../do-release-upgrade:33
msgid "Show version and exit"
msgstr "Näytä versio ja poistu"

#: ../update-manager:59
msgid "Directory that contains the data files"
msgstr "Datatiedostoja sisältävä hakemisto"

#: ../update-manager:62
msgid "Check if a new Ubuntu release is available"
msgstr "Tarkista onko uutta Ubuntu-julkaisua saatavilla"

#: ../update-manager:65 ../do-release-upgrade:36 ../check-new-release:27
#: ../check-new-release-gtk:149
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Tarkista onko päivittäminen uusimpaan kehitysversioon mahdollista"

#: ../update-manager:69
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Päivitä käyttäen uusinta ehdotettua julkaisupäivitysohjelmaa"

#: ../update-manager:76
msgid "Do not focus on map when starting"
msgstr "Älä kohdista karttaan käynnistettäessä"

#: ../update-manager:79
msgid "Try to run a dist-upgrade"
msgstr "Yritä ajaa jakelupäivitys (dist-upgrade)"

#: ../update-manager:83 ../do-release-upgrade:52
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Testaa päivitystä hiekkalaatikossa aufs-peitekerroksella"

#: ../update-manager:103
msgid "Running partial upgrade"
msgstr "Suoritetaan osittaista päivitystä"

#: ../do-release-upgrade:40 ../check-new-release:31
#: ../check-new-release-gtk:153
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Yritä päivittää uusimpaan julkaisuun käyttäen päivitysohjelmaa kohteesta "
"$distro-proposed"

#: ../do-release-upgrade:44
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Suorita erityisessä päivitystilassa.\n"
"Tällä hetkellä tuettuina ovat \"desktop\" työpöytäjärjestelmän ja \"server\" "
"palvelinjärjestelmän päivittämiseen."

#: ../do-release-upgrade:50
msgid "Run the specified frontend"
msgstr "Aja määritetty edustaohjelma"

#: ../do-release-upgrade:55
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Tarkista vain onko uusi jakelujulkaisu saatavilla, ja kerro tulos "
"poistumiskoodilla"

#: ../do-release-upgrade:69
msgid "Checking for a new ubuntu release"
msgstr "Tarkistetaan onko uutta Ubuntu-julkaisua saatavilla"

#: ../do-release-upgrade:79
msgid "No new release found"
msgstr "Uutta julkaisua ei löytynyt"

#: ../do-release-upgrade:85 ../check-new-release:56
#, c-format
msgid "New release '%s' available."
msgstr "Uusi julkaisu ”%s” saatavilla."

#: ../do-release-upgrade:86 ../check-new-release:57
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Aja \"do-release-upgrade\" päivittääksesi siihen."

#: ../check-new-release:35
#, c-format
msgid ""
"Do not output anything, just return %s if there is a new release and %s if "
"there is none."
msgstr ""
"Älä tulosta mitään, palauta vain %s jos uusi julkaisu on saatavilla tai %s "
"jos ei ole."

#: ../check-new-release-gtk:67
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Päivitys versioon Ubuntu %(version)s on saatavilla"

#: ../check-new-release-gtk:103
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Voit valita päivityksen Ubuntun versioon %s"

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Toteuttamaton metodi: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Tiedosto levyllä"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb-paketti"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Asenna puuttuva paketti."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Paketti %s tulisi asentaa."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i vanhentunutta tietoa tilatiedostossa"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "Vanhentuneita kohtia dpkg-tilatiedoissa"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "Vanhentuneita dpkg-tilatietoja"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Jos paketti kdelibs4-dev on asennettu, on päivityksen aikana oltava "
"asennettuna myös paketti kdelibs5-dev. Lisätietoja löytyy osoitteesta "
"bugs.launchpad.net löytyvästä virheraportista #279621."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s tulee merkitä käsin asennetuksi."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Poista lilo, koska grub on myös asennettu (katso bugista #314004 lisätietoja)"

#~ msgid ""
#~ "Canonical Ltd. no longer provides support for the following software "
#~ "packages. You can still get support from the community.\n"
#~ "\n"
#~ "If you have not enabled community maintained software (universe), these "
#~ "packages will be suggested for removal at the end of the upgrade."
#~ msgstr ""
#~ "Canonical Ltd. ei enää tue seuraavia ohjelmapaketteja. Yhteisö tukee niitä "
#~ "edelleen.\n"
#~ "\n"
#~ "Jos et ole ottanut käyttöön yhteisön ylläpitämien sovelluksien "
#~ "ohjelmalähdettä (\"universe\"), suositellaan nämä paketit  poistettaviksi "
#~ "päivityksen lopuksi."

#, python-format
#~ msgid ""
#~ "This session appears to be running under ssh. It is not recommended to "
#~ "perform a upgrade over ssh currently because in case of failure it is harder "
#~ "to recover.\n"
#~ "\n"
#~ "If you continue, a additional ssh daemon will be started at port '%s'.\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Tämä istunto näyttää toimivan SSH:n avulla. Päivityksen teko SSH:n yli ei "
#~ "ole suositeltavaa, koska ongelmien ilmaantueessa niistä on vaikeampi "
#~ "toipua.\n"
#~ "\n"
#~ "Jos jatkat, uusi SSH-palvelin käynnistetään porttiin \"%s\". \n"
#~ "\n"
#~ "Haluatko jatkaa?"
